templates/shared/forms/interactive/managed-kafka.html

```diff
@@ -0,0 +1,205 @@
+<div class="p-modal" id="contact-modal">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+    <header class="p-modal__header" style="display: block; border-bottom: 0;">
+      <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
+      <hr />
+      <h2 class="p-modal__title" id="modal-title">Canonical’s Managed Kafka</h2>
```

Comment:
Can you add the class `u-sv1` to this header for all interactive forms and the creation script, please.

Reply:
on the H2... yes